apps/oozie/src/oozie/templates/editor2/coordinator_editor.mako

```diff
@@ -51,7 +51,7 @@ ${ commonheader(_("Coordinator Editor"), "Oozie", user, request) | n,unicode }
 
     <a title="${ _('Save') }" rel="tooltip" data-placement="bottom" data-loading-text="${ _("Saving...") }"
         data-bind="click: $root.save, css: {'btn': true, 'disabled': $root.isSaving()}, visible: coordinator.properties.workflow() && canEdit">
-      <i class="fa fa-save"></i>
+      <i class="fa fa-save" data-testid="save-as-button"></i>
```

Comment:
This is in the incorrect file.

Once you find the correct place it should be "editor--save-as--button"